packages/edge-bundler/node/manifest.test.ts

```diff
@@ -1,7 +1,7 @@
-import { env } from 'process'
-
 import { test, expect } from 'vitest'
 
+// @ts-expect-error current tsconfig.json doesn't allow this, but I don't want to change it
+import { version } from '../package.json' assert { type: 'json' }
```

Comment:
Can we use import assertions? I don't believe so in the range of supported node versions?

Reply:
This is a test file, and CI doesn't show any errors because of it. So I think we can use it!